migrations/sql/R__view_mine_summary.sql

```diff
@@ -17,53 +23,133 @@ AS SELECT m.mine_guid::character varying AS mine_guid,
     mos.description AS mine_operation_status_d,
     mosr.description AS mine_operation_status_reason_d,
     mossr.description AS mine_operation_status_sub_reason_d,
-    m.create_timestamp ::character varying AS mine_date,
-    ms.effective_date::character varying AS status_date,
-    array_to_string(array_agg(DISTINCT mttc.description), ','::text) AS tenure,
-    array_to_string(array_agg(DISTINCT mttc.mine_tenure_type_code), ','::text) AS tenure_type_code,
-    array_to_string(array_agg(DISTINCT mcc.description), ','::text) AS commodity,
-    array_to_string(array_agg(DISTINCT mcc.mine_commodity_code), ','::text) AS commodity_type_code,
-    array_to_string(array_agg(DISTINCT mdc.description), ','::text) AS disturbance,
-    array_to_string(array_agg(DISTINCT mdc.mine_disturbance_code), ','::text) AS disturbance_type_code,
-    m.major_mine_ind::character varying AS major_mine_ind,
-    m.mine_region,
-    p.permit_no,
+    m.create_timestamp::text AS mine_date,
+    ms.effective_date::text AS status_date,
+    
+    array_to_string(array_agg(DISTINCT mttc.description), ','::text) AS mine_tenure,
+    array_to_string(array_agg(DISTINCT mttc.mine_tenure_type_code), ','::text) AS mine_tenure_type_code,
+    array_to_string(array_agg(DISTINCT mcc.description), ','::text) AS mine_commodity,
+    array_to_string(array_agg(DISTINCT mcc.mine_commodity_code), ','::text) AS mine_commodity_type_code,
+    array_to_string(array_agg(DISTINCT mdc.description), ','::text) AS mine_disturbance,
+    array_to_string(array_agg(DISTINCT mdc.mine_disturbance_code), ','::text) AS mine_disturbance_type_code,
+    
+    array_to_string(array_agg(DISTINCT mttc2.description), ','::text) AS permit_tenure,
+    array_to_string(array_agg(DISTINCT mttc2.mine_tenure_type_code), ','::text) AS permit_tenure_type_code,
+    array_to_string(array_agg(DISTINCT mcc2.description), ','::text) AS permit_commodity,
+    array_to_string(array_agg(DISTINCT mcc2.mine_commodity_code), ','::text) AS permit_commodity_type_code,
+    array_to_string(array_agg(DISTINCT mdc2.description), ','::text) AS permit_disturbance,
+    array_to_string(array_agg(DISTINCT mdc2.mine_disturbance_code), ','::text) AS permit_disturbance_type_code,
+    
+    m.mine_region AS mine_region,
+    m.major_mine_ind::text AS major_mine_ind,
+    '' AS bcmi_url,
```

Comment:
I guess they asked for this but we don't have it?

Reply:
This was in the view previously. I checked the code and yeah it looks like we haven't done anything with this.